pkg/utils/utils_test.go

```diff
@@ -85,10 +87,139 @@ func TestCustomizeLibertyEnv(t *testing.T) {
 	if err := verifyTests(testEnv); err != nil {
 		t.Fatalf("%v", err)
 	}
+}
+
+func TestCustomizeEnvSSO(t *testing.T) {
+	logf.SetLogger(logf.ZapLogger(true))
+	os.Setenv("WATCH_NAMESPACE", namespace)
+	svc := &openlibertyv1beta1.OpenLibertyApplicationService{Port: 8080, Type: &clusterType}
+	spec := openlibertyv1beta1.OpenLibertyApplicationSpec{Service: svc}
 
+	terminationPolicy := v1.TLSTerminationReencrypt
+	expose := true
+	spec.Env = []corev1.EnvVar{
+		{Name: "SEC_TLS_TRUSTDEFAULTCERTS", Value: "true"},
+		{Name: "SEC_IMPORT_K8S_CERTS", Value: "true"},
+	}
+	spec.Expose = &expose
+	spec.Route = &v1beta1.OpenLibertyApplicationRoute{
+		Host:        "myapp.mycompany.com",
+		Termination: &terminationPolicy,
+		Certificate: &v1beta1.Certificate{},
+	}
+	spec.SSO = &v1beta1.OpenLibertyApplicationSSO{
```

Comment:
Please add common parameters that apply to all providers: `sso.redirectToRPHostAndPort` and `sso.mapToUserRegistry`